admin_guide/manage_authorization_policy.adoc

```diff
@@ -602,8 +603,8 @@ rules:
   - builds
   verbs:
   - get
-	- list
-	- watch
+  - list
+  - watch
 ----
 
 Then, to use the current project, run:
```

Comment:
Are you changing to the project that you want to apply the role to? If so, this line isn't clear.

Reply:
I don't think so. This was existing content, and from what I understand it's adding a role to an existing project.

Reply:
That's fair, but it's still not super clear why it's the "current" project.

Reply:
I tried to clarify it a bit with the latest push :)

Reply:
That sounds better. Thank you! :fireworks: